app/statistic_class.py

```diff
@@ -0,0 +1,63 @@
+import operator
+from typing import List, Optional
+
+from app.base import SharedSettings
+from app.dynamodb_dao import DynamoDBDao
+from app.models import PetStatistics
+
+
+class StatisticsSettings(SharedSettings):
+    s3_bucket_name: str
+    days: Optional[int]
+    email_title: str
+
+
+class Statistic:
+    def __init__(
+        self,
+        dynamodb_table,
+        s3_client,
+        ses_service,
+        settings: StatisticsSettings,
+        dynamodb_dao: DynamoDBDao,
```

Comment:
remove this parameter, pass dynamodb_table to statistics and create DynamoDB dao from it